pages/get_involved.md

```diff
@@ -67,7 +63,7 @@ Most of the time we work with the consensus building techniques documented below
 
 ### Lazy Consensus
 
-[Lazy consensus][7] is the first, and possibly the most important, consensus building 
+[Lazy consensus][6] is the first, and possibly the most important, consensus building 
```

Comment:
No need to renumber. The numbers don't matter. They are just identifiers, and it's okay to skip one. If you want to, instead of numbers, you can give them names, such as:

```
[Lazy consensus][lazy] is the first...
.
.
.
[lazy]: ...
```

Reply:
Thanks for the additional info. The linking format is new to me so additional info helps. Not sure about the additional copy of the file. I found the dead link as I was parsing through the various contributor info related pages. As I begin work on ACCUMULO-4714 I'll try to track and address issues like this. Thanks for the review.

Reply:
I will merge this in and switch from numbers to identifiers to follow @ctubbsii suggestion.  I will post the commit so you get an idea of what we are talking about

Reply:
See d8338cc96c5a0eec